static/js/services/select2-search.js

```diff
@@ -88,17 +89,44 @@ angular.module('inboxServices').factory('Select2Search',
           });
       };
 
-      var resolveInitialValue = function(selectEl) {
+      var resolveInitialValue = function(selectEl, initialValues) {
+        // TODO: add support for multiples values here
```

Comment:
I didn't want to fix or raise because YAGNI. I wanted notes here because I wanted to show what was constraining multiple values, which I consider important because we support select2 selects that support multiple values, just not initialising them with multiple values.

What I will do is rework the function so that is only takes one value, not an array of one. That should make the intention clear without needing TODOs.